### DISCUSSION TITLE
How to have more than 10 pictures when I make a request?

### DISCUSSION THREAD

**Discussion by Bidulle59** [yurijmikhalevich/rclip#136] [Q&A] [answered]
Hi,
It works really fine.
I don't find how I can request to have more than 10 pictures in the response of my prompt.
Perhaps it's a stupid question. Sorry for that

**Comment by yurijmikhalevich** [comment, marked as answer]
Hi @Bidulle59! Thank you for the great feedback! I am glad you like rclip. You can pass the `-t N` param to get top-N pictures in the results. For example, to get top-100 pictures, do this:

```sh
rclip -t 100 cat
```

Let me know if this is what you were looking for!

You can find out about other rclip features via `rclip --help` :)

**Reply by Bidulle59** [reply]
Thanks a lot.
It's exactly what I need.
I am a windows user, and was trying rclip /?
Thanks for your help

**Reply by yurijmikhalevich** [reply]
@Bidulle59, perfect. Of course, I am happy to help. Let me know if you have any other questions or suggestions 😄 

> I am a windows user, and was trying rclip /?

I see! It can be worth considering mapping `/?` to `--help` to reduce confusion 🤔
